app/components/Article/article.css

```diff
@@ -154,17 +154,17 @@ article .link-popup p {
   gap: var(--spacing-32);
 }
 
+.form {
+  left: 845px;
+}
+
 article .footer-comtainer {
   display: flex;
   align-items: center;
   margin-top: var(--spacing-56);
   margin-bottom: var(--spacing-24);
+  gap: 16px;
```

Comment:
try to avoid raw pixel values. In `root.css` there's a bunch of available variables that should be used for sizing and coiours. You use them as `var(<var name)`, like in the previous lines. So in this case it'd be `gap: var(--spacing-16);`. This is important both for consistency (as it's easier to change everything at once), but also because the mobile version often has subtle differences in spacing which should be reflected by this

Reply:
Agreed